### ISSUE TITLE
How to install in linux ?

### ISSUE THREAD

**Issue by nahush2321** [Tremeschin/ModularMusicVisualizer#17]
i id read about the python poetry that you suggested and i did according to that but how to go at a ui

**Comment by Tremeschin** [comment]
Current code in the default branch lacks functionality, the first link on the readme (right below the logo image) points to the previous branch where the two architectures old code is and actually do have functionality, have rendered videos on.

It did not have a GUI, was 100% scripting, tho the shaders are somewhat easily swappable on the old python preset script. Also it was very spaghetti code so use it at your own risk :p I got it running myself the other day so should work for you.

Will take a while for me to achieve functionality again plus a GUI since I'm very stuck in implementing a node editor backend, will have binaries for each platform as well so installing and running will be trivial.


TLDR; Project is on a weird stagnation state with many divergent code, now rewriting the actual functionality, check previous master branch for the code that works but no UI.
